stage1/tasks/hangman/README.md

```diff
@@ -0,0 +1,88 @@
+- At the beginning of the game, the question-answer pair should be chosen randomly.
+- The application can be done in English or in Russian language.
+
+## Repository requirements
+- The task should be done in private school's repository.
+- The code should be committed to a separate branch named as the task.
+- The `main` (or old style master) branch should be empty (contain only files like README.md or .gitignore).
+- Commit messages should follow the [guideline](https://docs.rs.school/#/git-convention)
+- Once the work is finished, create a pull request from a **development** branch to **main** (or master).
+- The pull request name should contain **the task name**.
+- The pull request description should be provided according to the [template](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr).
+
+## Technical requirements
+- The application is displayed correctly and functions properly in the latest version of Google Chrome.
+- It is **not allowed** to use:
+    - CSS frameworks (e.g., `Bootstrap`),
+    - JS frameworks (e.g., `Angular`, `React`, `Vue`, etc.),
+    - Outdated libraries (e.g. `JQuery`, etc.),
+    - `TypeScript`.
+- You can use CSS preprocessors (`SASS`, `SCSS`) and `normalize.css`.
+- Using JS frameworks (e.g., Angular, React, Vue, etc.) is not allowed.
+
+## CrossCheck Criteria (150 points)
+*It is recommended to print the secret word in the browser's console to facilitate the cross-check process.*
+- [ ] Responsive/adaptive UI from 1440px to 360px viewport: `+10`
+- [ ] The generation of DOM elements is implemented. `body` in the `index.html` is empty (can contain only `script` tag): `+20`
+- [ ] The game starts with the correct default view (empty gallows, blank lines for secret word, etc.) and a random question: `+5`
+- [ ] The user can play the game by using the virtual keyboard: `+20`
+- [ ] The user can play the game by using the physical keyboard: `+20`
+- [ ] When the letter is correct, it appears instead of the corresponding blank line. If the letter repeats in the word, all corresponding blank lines should be replaced by it: `+15`
+- [ ] When the letter is incorrect:
+    - [ ] the incorrect guesses counter is updated: `+5`
+    - [ ] a body part is added to the gallows: `+10`
+- [ ] The clicked/pressed letter is disabled: `+5`
+- [ ] The body parts appear on the gallows in the logical order (head, body, left arm, right arm, left leg, right leg): `+5`
+- [ ] When the user runs out of 6 attempts or wins the game, the modal window appears: `+10`
+- [ ] The modal window includes the message about the game's outcome (winning or losing), the secret word and the 'play again' button: `+10`
+- [ ] When the user clicks the 'play again' button, the game starts over by showing a new question and resetting the gallows, the incorrect guesses counter and the blank lines for the secret word: `+15`
```

Comment:
_The same suggestion about the word "underscores"_

**Suggested:** - [ ] When the user clicks the 'play again' button, the game starts over by showing a new question and resetting the gallows, the incorrect guesses counter and the underscores for the secret word: `+15`

Reply:
Done